process_data.sh

```diff
@@ -150,6 +161,25 @@ compute_snr_cnr(){
 
 
 
+compute_ghosting()
+{
+  local path_data="$1"
+  local path_processed_data="$2"
+  local subject="$3"
+  local session="$4"
+  local acq="$5"
+  local rec="$6"
+  # Create ghosting mask only on navigatd data
+  if [[ $rec == "rec-navigated" ]]; then
+    echo "Creating ghosting mask for ${subject} ${session} ${acq}"
+    ./../../../../create_ghosting_mask.py $path_data $path_processed_data $subject $session $acq
```

Comment:
Is there a cleaner way to call `create_ghosting_mask.py`? (i.e. "${path_processed_data}/create_ghosting_mask.py")

Same comment for "compute_ghosting.py"